google-oauth-client/pom.xml

```diff
@@ -93,5 +93,9 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.http-client</groupId>
+      <artifactId>google-http-client-gson</artifactId>
+    </dependency>
```

Comment:
There's already an entry for google-http-client-gson on line 82-86 with test scope - can drop the test scope

Reply:
oh missed that, thanks!